cli_reference/basic_cli_operations.adoc

```diff
@@ -72,24 +86,60 @@ The following table describes the common `osc` operations and their general synt
 |`delete`
 a|`osc delete -f _<file_path>_`
 
-`osc delete _<object_type>_ _<object_id>_`
+`osc delete _<object_type>_ _<object_name_or_id>_`
 
 `osc delete _<object_type>_ -l _<label>_`
-.^|Deletes the specified OpenShift object. An object configuration can also be passed in through STDIN.
+
+`osc delete all -l _<label>_`
+.^|Deletes the specified OpenShift object. An object configuration can also be passed in through STDIN. The `osc delete all -l _<label>_` operation deletes all objects matching the specified `_<label>_`.
 
 |`log`
 |`osc log -f _<pod_name>_ _<container_name>_`
 |Retrieves the log output for a specific pod or container. This command does not work for other object types.
 |===
 
-== Project CLI Operations
-These advanced operations are used to define and instantiate OpenShift objects at the project level.
+== Object Types
+OpenShift supports the following object types, some of which have abbreviated syntax that you can use with `osc`.
+
+.Supported Object Types
+[options="header"]
+|===
+
+|Object Type |Abbreviated Version
+
+|`build` |
+|`buildConfig` | `bc`
+|`deploymentConfig` | `dc`
+|`imageStream` | `is`
+|`imageStreamTag` | `istag`
+|`imageStreamImage` | `isimage`
+|`event` |`ev`
+|`minion` |`mi`
+|`pod` |`po`
+|`replicationController` |`rc`
+|`service` |`svc`
+|===
+
+== Project Operations
+These advanced operations for administrators are used to define and instantiate OpenShift objects at the project level.
 
 The simplest way to create a new project is:
-`osadm new-project _<project-name> --display-name=<display-name> --description=<description> --admin=<admin-username>`
 
+****
+`$ osadm new-project _<project_name>_ --display-name=_<display_name>_ --description=_<description>_ --admin=_<admin_username>_`
+****
+
+For example:
+
+====
 
-For example: `osadm new-project test --display-name="OpenShift 3 Sample" --description="This is an example project to demonstrate OpenShift v3" --admin=anypassword:test-admin` creates a new project called `test` that appears in the web console as "Openshift 3 Sample", with `test-admin` as the project admin.
+[options="nowrap"]
+----
+$ osadm new-project test --display-name="OpenShift 3 Sample" --description="This is an example project to demonstrate OpenShift v3" --admin=anypassword:test-admin`
+----
+====
+
+This creates a new project called `test` that appears in the Management Console as "Openshift 3 Sample", with `test-admin` as the project admin.
 
```

Comment:
Again, use the Example block.

The following example creates a new project called `test` that appears in the Management Console as "Openshift 3 Sample", with `test-admin` as the project admin.

[Example]
screen output